docs/api/airgun.entities.rst

```diff
@@ -0,0 +1,5 @@
+:mod:`airgun.entities`
+======================
+
+.. automodule:: airgun.entities
+   :members:
```

Comment:
Existing entities should probably be included in this file (similar to https://github.com/SatelliteQE/robottelo/blob/master/docs/api/robottelo.ui.rst)

Reply:
I had some solution with this already prepared:

```
docs/
├── api
│   ├── airgun.browser.rst
│   ├── airgun.entities.rst
│   ├── airgun.fixtures.rst
│   ├── airgun.navigation.rst
│   ├── airgun.session.rst
│   ├── airgun.settings.rst
│   ├── airgun.views.rst
│   ├── airgun.widgets.rst
│   ├── entities
│   │   ├── airgun.entities.architecture.rst
│   │   ├── airgun.entities.location.rst
│   │   ├── airgun.entities.login.rst
│   │   ├── airgun.entities.organization.rst
│   │   └── airgun.entities.os.rst
│   ├── index.rst
│   └── views
│       ├── airgun.views.architecture.rst
│       ├── airgun.views.login.rst
│       └── airgun.views.os.rst
├── build-docs.rst
├── conf.py
├── examples.rst
├── index.rst
└── Makefile
```

Would that work? I have no issues changing it.

Reply:
I think that's overkill - connecting 1 entity or view is about 3 lines, by creating separate files we'll end up with 60-80 almost empty files. I'd rather vote for 1 file for entities and 1 file for views.

Reply:
Agreed, done.